docbook5-book/xml/common_intro_available_doc.xml

```diff
@@ -22,43 +22,49 @@
   </dm:docmanager>
  </info>
 
- <note>
-  <title>Online documentation and latest updates</title>
-  <para>
-   Documentation for our products is available at
-   <link os="sles;sled" xlink:href="&dsc;"/><link os="osuse"
-   xlink:href="http://doc.opensuse.org/"/>,
-   where you can also find the latest updates, and browse
-   or download the documentation in various formats. The latest
-   documentation updates can usually be found in the English
-   language version.
-  </para>
- </note>
- <para>
-  In addition, the product documentation
-  is usually available in your installed system under
-  <filename>/usr/share/doc/manual</filename>.
- </para>
- <para>
-  The following documentation is available for this product:
- </para>
-
  <variablelist>
   <varlistentry>
-   <term><xref linkend="book-example"/></term>
+   <term>Online documentation</term>
+   <listitem>
+    <para>
+     The online documentation for this product is available at
+     <link xlink:href="&doc-url;"/>. Browse or download the documentation
+     in various formats.
+    </para>
+    <para condition="suse-product">
+     Find the online documentation for other products at
+     <link xlink:href="&dsc;"/>.
+    </para>
+    <note>
+     <title>Latest updates</title>
+     <para>
+      The latest updates are usually available in the English
+      language version of this documentation.
+     </para>
+    </note>
+   </listitem>
+  </varlistentry>
```

Comment:
@tomschr : I'll have a look next week when I'm back. I also need to apply some modifications from my (also WIP) https://github.com/SUSE/doc-sle/pull/1420 here because I found out that the &doc-url; entity cannot be used in this context. Therefore please do not merge this yet.

Reply:
@tomschr : In order not to blow up the intros again I decided against mentioning susedoc.github.io here.